.github/workflows/convert.yml

```diff
@@ -14,34 +14,143 @@ env:
-    if: github.repository_owner == 'dragonflyoss'
     steps:
       - name: Checkout repository
-        uses: actions/checkout@v2
-      - name: Install Nydus binaries
-        run: |
-          NYDUS_VERSION=$(curl --silent "https://api.github.com/repos/dragonflyoss/image-service/releases/latest" | grep -Po '"tag_name": "\K.*?(?=")')
-          wget https://github.com/dragonflyoss/image-service/releases/download/$NYDUS_VERSION/nydus-static-$NYDUS_VERSION-linux-amd64.tgz
-          tar xzf nydus-static-$NYDUS_VERSION-linux-amd64.tgz
-          sudo cp nydus-static/nydusify nydus-static/nydus-image /usr/local/bin/
-          sudo cp nydus-static/nydusd /usr/local/bin/nydusd
-      - name: Log in to the container registry
-        uses: docker/login-action@f054a8b539a109f9f41c372932f1ae047eff08c9
-        with:
-          registry: ${{ env.REGISTRY }}
-          username: ${{ github.actor }}
-          password: ${{ secrets.GITHUB_TOKEN }}
+        uses: actions/checkout@v3
       - name: Build fsck.erofs
         run: |
           sudo apt-get update && sudo apt-get install -y build-essential git autotools-dev automake libtool pkg-config uuid-dev liblz4-dev
           git clone https://git.kernel.org/pub/scm/linux/kernel/git/xiang/erofs-utils.git
           cd erofs-utils && git apply ../${{ env.FSCK_PATCH_PATH }} && ./autogen.sh && ./configure && make && cd ..
           sudo cp erofs-utils/fsck/fsck.erofs /usr/local/bin/
-      - name: Convert RAFS v5 images
+      - name: Upload fsck.erofs
+        uses: actions/upload-artifact@master
+        with:
+          name: fsck-erofs-artifact
+          path: |
+            /usr/local/bin/fsck.erofs
+
+  convert-zran:
+    runs-on: ubuntu-latest
+    needs: [nydusify-build, nydus-build]
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+      - name: Login ghcr registry
+        uses: docker/login-action@v2
+        with:
+          registry: ${{ env.REGISTRY }}
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+      - name: Download Nydus
+        uses: actions/download-artifact@master
+        with:
+          name: nydus-artifact
+          path: /usr/local/bin
+      - name: Download Nydusify
+        uses: actions/download-artifact@master
+        with:
+          name: nydusify-artifact
+          path: /usr/local/bin
+      - name: Convert and check zran images
         run: |
+          sudo chmod +x /usr/local/bin/nydus*
+          sudo docker run -d --restart=always -p 5000:5000 registry
+          for I in $(cat ${{ env.IMAGE_LIST_PATH }}); do
+            echo "converting $I:latest to $I:nydus-nightly-oci-ref"
+            # for pre-built images
+            sudo DOCKER_CONFIG=$HOME/.docker nydusify convert \
+                 --oci-ref \
+                 --source $I:latest \
+                 --target ${{ env.REGISTRY }}/${{ env.ORGANIZATION }}/$I:nydus-nightly-oci-ref \
+                 --fs-version 5
```

Comment:
Should we use v6 for zran?

Reply:
Thanks, fixed, we still enforce to use v6 for `--oci-ref` image.